11-test-react-mastery/src/components/App.css

```diff
@@ -1,5 +1,7 @@
 body {
   background-image: url("https://i.ibb.co/rcyn0rs/ro-background.jpg") !important;
+  object-fit: fill !important;
```

Comment:
https://stackoverflow.com/questions/3706819/what-are-the-implications-of-using-important-in-css

Read this and refactor accordingly, this is a good time to research why you need to apply !important

Reply:
I was able to remove the important tags, but is this okay? 
The codes from the 3rd party library were crossed out, which is similar when implementing !important.

![123](https://user-images.githubusercontent.com/32013541/163091123-3b209829-abb6-460f-b381-4191cdf9f771.PNG)

Reply:
Code*, can you link and explain to me what you did differently in this PR? Explore the UI of the "Files changed" tab and comment accordingly.